### DISCUSSION TITLE
Oqtane Team

### DISCUSSION THREAD

**Discussion by hishamco** [oqtane/oqtane.framework#885] [General]
Happy to ship Oqtane `2.0.0`, I'd like to take the opportunity and post few topics that may organize Oqtane project and probably increase the community around it

First let us discuss about the Oqtane Core members, as all knows @sbwalker is the only maintainer of this project, he likes most of us doing reviews, contribution on his spare time, but frankly this may reduce the interactivity to respond to the issues, review PRs .. etc. I already see something similar in other OSS that I worked on, one of them is Orchard Core, which is now has many members that take the responsibility to grow the project and move it forward

What I'm suggesting if @sbwalker don't mind is to select active members probably we could start with (2-4) then we could see how everything is going on, meanwhile we could see how active they are, because this is a responsibility of each one of them to move the project to the next level

Hope to hear from the community devs and contributors who love this project

Thanks

**Comment by mikecasas** [comment]
Just wanted to say congrats to everyone for the 2.0 release, especially @sbwalker and @hishamco for all their hard work.

**Reply by hishamco** [reply]
You too @mikecasas and all other contributors

**Comment by hishamco** [comment]
@sbwalker can we consider this ASAP if you don't mind, I see the repo is not active as expected, IMHO it would be nice to add two or three active users as Oqtane team members, this will allow us to accelarate the process of answering the issues, submitting the PR and implement new features

@chlupac is one of candidates and active members that I know, but we can add members accoring their contributions and commitments to make this project forward

**Comment by PizzaConsole** [comment]
Lots of good work! Thanks!
